src/rs.cpp

```diff
@@ -3430,3 +3431,213 @@ rs2_raw_data_buffer* rs2_terminal_parse_response(rs2_terminal_parser* terminal_p
 }
 HANDLE_EXCEPTIONS_AND_RETURN(nullptr, terminal_parser, command, response)
 
+void rs2_project_point_to_pixel(float pixel[2], const struct rs2_intrinsics* intrin, const float point[3]) BEGIN_API_CALL
```

Comment:
Why here? I suggest you move them to a new `rsutil.cpp`.

Reply:
The implementation for all public APIs is in a single file

Reply:
If we see fit to separate in a header, we can separate in a different .cpp